tap_facebook/__init__.py

```diff
@@ -219,11 +224,6 @@ def __attrs_post_init__(self):
         if self.options.get('primary-keys'):
             self.key_properties.extend(self.options['primary-keys'])
 
-    @backoff.on_exception(
-        backoff.expo,
-        (InsightsJobTimeout),
-        max_tries=3,
-        factor=2)
```

Comment:
This function doesn't have a throw for the `InsightsJobTimeout`, so that's why the decorator was removed.